android/src/main/java/io/ably/flutter/plugin/push/PushActivationEventHandlers.java

```diff
@@ -64,13 +64,23 @@ public void onReceive(Context context, Intent intent) {
       switch (action) {
         case PUSH_ACTIVATE_ACTION:
           callCallbackOnDartSide(PlatformConstants.PlatformMethod.pushOnActivate, errorInfo);
-          returnMethodCallResult(resultForActivate, errorInfo);
-          resultForActivate = null;
+          if (resultForActivate != null){
+            Log.d(TAG, "resultForActivate received on PUSH_ACTIVATE_ACTION.");
+            returnMethodCallResult(resultForActivate, errorInfo);
+            resultForActivate = null;
+          }else{
```

Comment:
Please run code auto-formatting as this line (and a few more) are missing whitespace :wink: 
```suggestion
          } else {
```

Reply:
Autoformatting will generally cause issues (lots of changes) on plugin code as there is no configuration (AFAIK) set up. I will try to manually format it

Reply:
Given there's no style configuration I wouldn't want to block you on manually formating the code :wink: Therefore, I'll resolve this comment and approve the PR as this is just a cosmetic change :+1: